api/client/inspect.go

```diff
@@ -57,20 +61,33 @@ func (cli *DockerCli) CmdInspect(args ...string) error {
 				continue
 			}
 		} else {
-			var value interface{}
-
 			// Do not use `json.Unmarshal()` because unmarshal JSON into
 			// an interface value, Unmarshal stores JSON numbers in
 			// float64, which is different from `json.Indent()` does.
```

Comment:
not sure about removing this comment also. @LK4D4 do you think this need dec.UseNumber() anymore? I don't think so, number from types should be handled correctly, or not?